README.md

```diff
@@ -123,13 +125,7 @@ method or passed through the ```.s3_sync``` configuration file.
 
 Alternatively, if you are running builds on EC2 instance which has approrpiate IAM role, then you don't need to think about specifying credentials at all – they will be pulled from AWS metadata service.
 
-#### Through the command line
-
-The aws credentials can also be passed via a command line options
-`--aws_access_key_id` and `aws_secret_access_key`. They should override
```

Comment:
You are commenting on a deleted line (red). If you look at my PR changes (green) you'll see that I changed it to --aws_secret_access_key